src/components/CitySelectionForm.js

```diff
@@ -0,0 +1,45 @@
+import "./../styles/CitySelectionForm.css";
+import { useState } from "react";
+
+function CitySelectionForm(props) {
+  const [userChoice, setUserChoice] = useState("placeholder");
+  // console.log("Inicial Users Choice State:", userChoice); //RETURNS placeholder  - OK
```

Comment:
Not sure if this is still being used, but if not then the commented code can be removed